Anagram/Anagram.cs

```diff
@@ -0,0 +1,85 @@
+﻿
+namespace Anagram
+{
+    public partial class Anagram : Form
+    {
+        private String[] wordList;
+        List<string> matchedWordsListFinal;
+        public Anagram()
+        {
+            InitializeComponent();
+            this.wordList = new String[] { };
+            this.matchedWordsListFinal = new List<string>();
+
+        }
+
+        public int LoadWords()
+        {
+            this.wordList = System.IO.File.ReadAllText(@"C:\Users\Krasskoo\Desktop\WordList.txt").Split(new char[] { '.', '?', '!', ' ', ';', ':', ',', '\n' }, StringSplitOptions.RemoveEmptyEntries);
+            return this.wordList.Length;
+        }
+        public int FillMatchedWords(int totalWords)
+        {
+            String inputString = inputWordBox.Text;
+            int inputStringLength = inputString.Length;
+
+            char[] inputStringChars = inputString.ToUpper().ToCharArray();
+            Array.Sort(inputStringChars);
+
+            String sortedInputString = new string(inputStringChars);
+            for (int m = 0; m < totalWords; m++)
```

Comment:
use the modern way to iterate a collection. this will save code, thus improve readability and spare redundant variables